src/client/index.ts

```diff
@@ -451,48 +417,6 @@ class Client extends EventEmitter {
   static encodeXAddress = encodeXAddress
   static decodeXAddress = decodeXAddress
 
-  /**
-   * Static methods that replace functionality from the now-deprecated ripple-hashes library
-   */
-  // Compute the hash of a binary transaction blob.
-  // @deprecated Invoke from top-level package instead
-  static computeBinaryTransactionHash = computeBinaryTransactionHash // (txBlobHex: string): string
-  // Compute the hash of a transaction in txJSON format.
-  // @deprecated Invoke from top-level package instead
-  static computeTransactionHash = computeTransactionHash // (txJSON: any): string
-  // @deprecated Invoke from top-level package instead
-  static computeBinaryTransactionSigningHash =
-    computeBinaryTransactionSigningHash // (txBlobHex: string): string
-  // Compute the hash of an account, given the account's classic address (starting with `r`).
-  // @deprecated Invoke from top-level package instead
-  static computeAccountLedgerObjectID = computeAccountLedgerObjectID // (address: string): string
-  // Compute the hash (ID) of an account's SignerList.
-  // @deprecated Invoke from top-level package instead
-  static computeSignerListLedgerObjectID = computeSignerListLedgerObjectID // (address: string): string
-  // Compute the hash of an order, given the owner's classic address (starting with `r`) and the account sequence number of the `OfferCreate` order transaction.
-  // @deprecated Invoke from top-level package instead
-  static computeOrderID = computeOrderID // (address: string, sequence: number): string
-  // Compute the hash of a trustline, given the two parties' classic addresses (starting with `r`) and the currency code.
-  // @deprecated Invoke from top-level package instead
```

Comment:
These should all be renamed to `compute...Index` as that's the term used by rippled

Reply:
@mvadari so would `computeOrderId` become `computerOrderIDIndex`, `computeOrderIndex` or something else? (Are there docs with those Rippled terms somewhere?)

Reply:
Replaced 'ID' with 'Index' for these functions (So `computeOrderId` became `computeOrderIndex`) - The docs referenced are the api calls on xrpl.org: e.g. https://xrpl.org/offer.html